client/stylesheets/_responsive.scss

```diff
@@ -115,6 +120,37 @@
     .profileSettingsLi h4 {
         font-size: 1rem !important;
     }
+
+    .profileUserName h1  {
+        font-size: 1.8rem !important;
+    }
+
+    .profileUserAmount ul li {
+        font-size: 0.8rem !important;
+    }
+
+    .box-title {
+        font-size: 1rem !important;
+    }
+
+    .profileChatBtn a  {
+        font-size: 1rem !important;
+        padding: 14px 18px !important;
+    }
+
+    .optionsProClick ul .clicker {
+        font-size: 1rem !important;
+        padding: 14px 18px !important;
+    }
+
+    .profileUserInfo > ul li:first-child {
+        font-size: 0.9rem !important;
+    }
+
+    .profileUserImage img {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `profileUserImage` should be written in lowercase with hyphens